gcloud/_helpers.py

```diff
@@ -30,9 +30,13 @@
 except ImportError:
     app_identity = None
 try:
-    from grpc.beta import implementations
+    from google.gax.grpc import exc_to_code as beta_exc_to_code
+    import grpc
+    from grpc._channel import _Rendezvous
```

Comment:
Fewer code elements in the API overall, fewer occurrences same-method-on-different-classes, and shorter textual length of application code.